### DISCUSSION TITLE
Agenda e circolari

### DISCUSSION THREAD

**Discussion by LTChamp** [Lioydiano/Classeviva#17] [Domande e Risposte] [answered]
Ciao, vorrei inserire nel mio sito fatto con python, javascript e Html, l'agenda e le circolari di Classeviva.
Ovviamente però l'agenda e le circolari devono essere sempre in aggiornamento per non far mancare mai nulla!
Come posso fare?

**Comment by FLAK-ZOSO** [comment]
Allora... per quanto riguarda l'agenda, ce n'è un virtuoso esempio in [Spaggiu](https://spaggiu.alenygam.com/) il cui sorgente si trova [qui](https://github.com/Alenygam/spaggiu).

Per quanto riguarda invece le circolari, le puoi ottenere altrettanto facilmente tramite Classeviva.py, tuttavia non penso abbia molto senso renderle accessibili tramite un servizio tuo, quando puoi direttamente reindirizzare l'utente al link di download reso disponibile da Classeviva stessa.

**Reply by LTChamp** [reply, marked as answer]
Grazie mille
